.github/workflows/bare.yaml

```diff
@@ -58,14 +60,25 @@ jobs:
         echo "${{ secrets.SFTPPLUS_BIN_HOST_KEY }}" > ~/.ssh/known_hosts
         choco install --yes --no-progress openssh
 
-    # Uploads using a (per-OS selected) sftp command, then shows final links.
+    # Upload using a (per-OS selected) sftp command, then show final links.
     - name: Upload testing package
-      run: bash ./publish_dist.sh
+      run: |
+        bash ./publish_dist.sh
+        rm priv_key
```

Comment:
I am not if `rm priv_key` is called when `bash` fails.

it might need a final cleanup step with `if: always()`

Reply:
Good catch, thanks! I can remove the key in the same line regardless of the error code from the publishing script.